src/loader/mod.rs

```diff
@@ -0,0 +1,580 @@
+// Copyright (c) 2019 Intel Corporation. All rights reserved.
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Copyright 2017 The Chromium OS Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE-BSD-3-Clause file.
+//
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
+
+extern crate vm_memory;
+
+use std::error::{self, Error as KernelLoaderError};
+use std::ffi::CStr;
+use std::fmt::{self, Display};
+use std::io::{Read, Seek, SeekFrom};
+use std::mem;
+
+use vm_memory::{Address, Bytes, GuestAddress, GuestMemory, GuestMemoryMmap, GuestUsize};
+
+#[allow(dead_code)]
```

Comment:
Why is this necessary?

Reply:
remove #[allow(dead_code)], is that what you mean?

Reply:
I was asking why do you need to have `#[allow(dead_code)]`? I am thinking that you needed it because bootparam.rs looks like an autogenerated file and you might not use all the defines there and you are also not re-exporting them. So probably if you don't allow dead code the build will be full of warnings. I just wanted to check that's the case. It was more of a question than a change request.

Reply:
Yep, you're quite right! just because bootparam.rs is autogenerated and contains a bunch of unused code, add this to remove the build warnings.